### ISSUE TITLE
Key InceptionResnetV1/Logits/Bottleneck/biases not found in checkpoint

### ISSUE THREAD

**Issue by davidmuggleton** [zZyan/race_gender_recognition#5]
Hi @zZyan,

Thanks so much for providing this, proofing very useful to study your implementation. I am however encountering an error when attempting to use the trainer (`trainer.py`):

```
Caused by op 'save_1/RestoreV2', defined at:
  File "trainer.py", line 226, in <module>
    num_epoch = args.num_epoch)
  File "trainer.py", line 153, in run
    input_saver = tf.train.Saver(variables_to_restore)
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/training/saver.py", line 832, in __init__
    self.build()
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/training/saver.py", line 844, in build
    self._build(self._filename, build_save=True, build_restore=True)
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/training/saver.py", line 881, in _build
    build_save=build_save, build_restore=build_restore)
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/training/saver.py", line 513, in _build_internal
    restore_sequentially, reshape)
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/training/saver.py", line 332, in _AddRestoreOps
    restore_sequentially)
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/training/saver.py", line 580, in bulk_restore
    return io_ops.restore_v2(filename_tensor, names, slices, dtypes)
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/ops/gen_io_ops.py", line 1572, in restore_v2
    name=name)
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/framework/op_def_library.py", line 788, in _apply_op_helper
    op_def=op_def)
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/util/deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/framework/ops.py", line 3300, in create_op
    op_def=op_def)
  File "/Users/m/Library/Python/3.7/lib/python/site-packages/tensorflow/python/framework/ops.py", line 1801, in __init__
    self._traceback = tf_stack.extract_stack()

NotFoundError (see above for traceback): Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Key InceptionResnetV1/Logits/Bottleneck/biases not found in checkpoint
	 [[node save_1/RestoreV2 (defined at trainer.py:153) ]]
```

I am attempting to restore from the `20180402-114759 Inception ResNet v1` pre-trained model. I've followed each step in order from your README. I was wondering if you could point me in the right direction in getting this working?

Many thanks!

**Comment by liuxn0811** [comment]
I've found a method that works well.
You can change:
`exclude = ['Added', 'Race', 'Gender', 'InceptionResnetV1/Bottleneck/biases']`
to
`exclude = ['Added', 'Race', 'Gender', 'InceptionResnetV1/Logits/Bottleneck/']`
Line 14 in multitask.py.

**Comment by davidmuggleton** [comment]
@liuxn0811 that worked! Amazing! Thank you!

**Comment by openedhardware** [comment]
@liuxn0811 @davidmuggleton 
Were you able to freeze the current checkpoint files to .pb file?

If so, could you share the correct output node name(s)?

Thanks!

**Comment by davidmuggleton** [comment]
@openedhardware what's the issue your encountering?
